### DISCUSSION TITLE
Change the behaviour of the terminal that opens when choosing a software from the startup menu

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#138] [General]
Currently we open a shell within the container. This is very confusing, because the shell will behave differently compared to a shell that is opened from the start-up menu + in every package container the shell will behave a bit differently (depending on the environment, what tools are installed within the container, etc.)

I suggest that when clicking on the menu, VNM will open a regular shell (the same one that opens when the user clicks on the "terminal" icon in the toolbar) and just load the specific package clicked upon in the GUI. This will give the user a more consistent behaviour, and will also allow more experienced users to load additional containers (in the same shell session) if necessary.

**Comment by stebo85** [comment]
yes, this could be done - but first the module system has to be hacked to enable a download of packages. I recommend you start working on issue #64 and once this is completed this could be done.
